bin/generate_npm_tarball.sh

```diff
@@ -1,21 +1,28 @@
 #!/bin/bash
 
-if [ $# -ne 2 ]; then
-  echo "usage: $0 PACKAGE[@VERSION] OUTPUT.TGZ"
+if [ $# -le 2 ]; then
+  echo "usage: $0 PACKAGE[@VERSION] OUTPUT.TGZ [USE NODEJS 6]"
   exit 1
 fi
 package=$1
 output=$(pwd)/$2
+usenodejs6=$3
```

Comment:
Probably good to have a default. You're using SH style comparisons (as opposed to bash) which means `[ $empty = true ]` will result in an error. Another way to avoid it is using bash comparisons: `[[ $empty == true ]]`.
```suggestion
usenodejs6=${3:-false}
```